xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-rss/src/main/java/org/xwiki/rendering/internal/macro/rss/RssMacro.java

```diff
@@ -69,6 +70,8 @@
 
     private static final String FEED_CLASS_VALUE = "rssfeed";
 
+    private static final String FEED_PROPERTY = "feed";
```

Comment:
"feed" have a big chance of colliding with another property from the ExecutionContext, would be safer to put something more explicit like "RSSMacro.feed".